tests/ui/inline-const/interpolated.rs

```diff
@@ -0,0 +1,25 @@
+// check-pass
+
+#![feature(inline_const)]
+
+macro_rules! m {
+    ($b:block) => {
+        fn foo() {
+            const $b
+        }
+    }
+}
+
+// This already worked
```

Comment:
worked when? please add an issue so this is clear to future readers and then r=me

Reply:
hmm actually there isn't an issue to link to 🤣 then idk if this comment would be useful

Reply:
maybe you could include a mention of this in your commit message so git archaeologists will still be able to find this info

Reply:
sorry, what I meant was "this has worked since `const {}` was implemented", to contrast it with the specific case that `const {}` is used in a statement position.

Reply:
I can add a comment tho.

Reply:
Yeah, I think a comment describing what this test does would be nice